Makefile

```diff
@@ -182,7 +189,7 @@ test_cover:
 
 test_lint:
 	gometalinter.v2 --config=tools/gometalinter.json ./...
-	!(gometalinter.v2 --exclude /usr/lib/go/src/ --exclude client/lcd/statik/statik.go --disable-all --enable='errcheck' --vendor ./... | grep -v "client/")
+	!(gometalinter.v2 --exclude /usr/lib/go/src/ --exclude client/lcd/statik/statik.go --exclude 'vendor/*' --disable-all --enable='errcheck' --vendor ./... | grep -v "client/")
```

Comment:
`--exclude 'vendor/*'` and `--vendor` seem redundant, no?

Reply:
They ought to be, but they're not. `--vendor` prevents it from being searched thus saving time. For some reason, theres a bug in metalinter where golang/net/addr still gets linting output that gets printed out. Thus the exclude is just not letting those be printed / change the error code.

Reply:
Ugh OK thanks